app/controllers/api/v1/classes_controller.rb

```diff
@@ -56,14 +56,22 @@ def render_info(clazz)
           :last_name => teacher.user.last_name
         }
       },
-      :students => clazz.students.includes(:user).map {|student|
+      :students => clazz.students.includes(:user, :learners).map {|student|
```

Comment:
Remove :learners - this was there because I was writing out the student endpoints here too but I removed it